README.md

```diff
@@ -95,40 +101,74 @@ Note: you could turn off your internet connection, and the script inference woul
 
 Type `exit` to finish the script.
 
-
 ### CLI
+
 The script also supports optional command-line arguments to modify its behavior. You can see a full list of these arguments by running the command ```python privateGPT.py --help``` in your terminal.
 
+## How does it work?
 
-# How does it work?
 Selecting the right local models and the power of `LangChain` you can run the entire pipeline locally, without any data leaving your environment, and with reasonable performance.
 
 - `ingest.py` uses `LangChain` tools to parse the document and create embeddings locally using `HuggingFaceEmbeddings` (`SentenceTransformers`). It then stores the result in a local vector database using `Chroma` vector store.
 - `privateGPT.py` uses a local LLM based on `GPT4All-J` or `LlamaCpp` to understand questions and create answers. The context for the answers is extracted from the local vector store using a similarity search to locate the right piece of context from the docs.
 - `GPT4All-J` wrapper was introduced in LangChain 0.0.162.
 
-# System Requirements
+## System Requirements
 
 ## Python Version
+
 To use this software, you must have Python 3.10 or later installed. Earlier versions of Python will not compile.
 
 ## C++ Compiler
+
 If you encounter an error while building a wheel during the `pip install` process, you may need to install a C++ compiler on your computer.
 
 ### For Windows 10/11
+
 To install a C++ compiler on Windows 10/11, follow these steps:
 
 1. Install Visual Studio 2022.
 2. Make sure the following components are selected:
-   * Universal Windows Platform development
-   * C++ CMake tools for Windows
+   - Universal Windows Platform development
+   - C++ CMake tools for Windows
 3. Download the MinGW installer from the [MinGW website](https://sourceforge.net/projects/mingw/).
 4. Run the installer and select the `gcc` component.
 
 ## Mac Running Intel
+
 When running a Mac with Intel hardware (not M1), you may run into _clang: error: the clang compiler does not support '-march=native'_ during pip install.
 
 If so set your archflags during pip install. eg: _ARCHFLAGS="-arch x86_64" pip3 install -r requirements.txt_
 
-# Disclaimer
+## Using GPU acceleration
+
+1. Install [NVidia CUDA 11.8](https://developer.nvidia.com/cuda-11-8-0-download-archive)
+2. Build `llama-cpp-python` package with cuBLAS enabled. Run the code below in the directory you want to build the package in.
+   - Powershell:
+
+   ```powershell
+   git clone https://github.com/abetlen/llama-cpp-python
+   cd llama-cpp-python/vendor
+   git clone https://github.com/ggerganov/llama.cpp
+   cd ..
+   pip3 install scikit-build
+   $Env:CMAKE_ARGS="-DLLAMA_CUBLAS=on"; $Env:FORCE_CMAKE=1; py ./setup.py install
```

Comment:
llama.cpp has support to OpenCL with LLAMA_CLBLAST, so for AMD cards (List of GPUs https://docs.amd.com/bundle/Hardware_and_Software_Reference_Guide/page/Hardware_and_Software_Support.html) installing ROCm and using LLAMA_CLBLAST=1 works

For older cards maybe the OpenCL implementation on AMDGPU may work.